micromanager_gui/main_window.py

```diff
@@ -112,9 +112,7 @@ def __init__(self, viewer: napari.viewer.Viewer, remote=True):
         # to core may outlive the lifetime of this particular widget.
         sig.sequenceStarted.connect(self._on_mda_started)
         sig.sequenceFinished.connect(self._on_mda_finished)
-        sig.sequenceFinished.connect(
-            self._refresh_options
-        )  # why when acq is finished?
+        sig.sequenceFinished.connect(self._refresh_options)  # why when acq is finished?
```

Comment:
following up on this comment - why is this happening? I think if the signals are set up correctly then the gui should stay up to date through the mda